contracts/Swap/SwapRouter.sol

```diff
@@ -283,26 +606,101 @@ contract SwapRouter is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, IPan
         }
     }
 
+    // **** SWAP (supporting fee-on-transfer tokens) ****
+    // requires the initial amount to have already been sent to the first pair
+    function _swapSupportingFeeOnTransferTokens(
+        uint256 amountIn,
+        address[] memory path,
+        address _to
+    ) internal virtual returns (uint256[] memory amounts) {
```

Comment:
I would say we are not using the returned value in any place. Could we remove it?

Reply:
Fixed. can verify at [commit](https://github.com/VenusProtocol/venus-protocol/pull/204/commits/fab8ab3067ed480ac4c834762a8a1160e9866af8).